specification/riskiq/Easm/routes.tsp

```diff
@@ -0,0 +1,166 @@
+  @action("validate")
+  validateDiscoGroup is Operations.ResourceCollectionAction<
+    DiscoGroup,
+    DiscoGroupData,
+    ValidateResult
+  >;
+  @doc("Retrieve a discovery group with a given groupName.")
+  getDiscoGroup is Operations.ResourceRead<DiscoGroup>;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Create a discovery group with a given groupName.")
+  @put
+  putDiscoGroup is Foundations.ResourceOperation<
+    DiscoGroup,
+    DiscoGroupData,
+    DiscoGroup
+  >;
+  @doc("Run a discovery group with a given groupName.")
+  @action("run")
+  runDiscoGroup is Operations.ResourceAction<DiscoGroup, {}, NoContentResult>;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Retrieve a collection of discovery run results for a discovery group with a given groupName.")
+  @route("/discoGroups/{groupName}/runs")
+  @get
+  listRuns is Foundations.Operation<
+    DiscoveryGroupsListRunsParameter,
+    DiscoRunPageResult
+  >;
+}
+
+@tag("Discovery Templates")
+@doc("Provides access to disco template resources.")
+interface DiscoveryTemplates {
+  @doc("Retrieve a list of disco templates for the provided search parameters.")
+  listDiscoTemplate is Operations.ResourceList<DiscoTemplate, CommonListTrait>;
+  @doc("Retrieve a disco template with a given templateId.")
+  getDiscoTemplate is Operations.ResourceRead<DiscoTemplate>;
+}
+
+@tag("Reports")
+@doc("Provides access to reporting data.")
+interface Reports {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Get billable assets summary for the workspace.")
+  @route("/reports/assets:getBillable")
+  @post
+  getBillable is Foundations.Operation<{}, ReportBillableAssetSummaryResult>;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Get the most recent snapshot of asset summary values for the snapshot request.")
+  @route("/reports/assets:getSnapshot")
+  @post
+  getSnapshot is Foundations.Operation<
+    ReportAssetSnapshotRequest,
+    ReportAssetSnapshotResult
+  >;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Get asset summary details for the summary request.")
+  @route("/reports/assets:getSummary")
+  @post
+  getSummary is Foundations.Operation<
+    ReportAssetSummaryRequest,
+    ReportAssetSummaryResult
+  >;
+}
+
+@tag("Saved Filters")
+@doc("Provides access to saved filter resources.")
+interface SavedFilters {
+  @doc("Retrieve a list of saved filters for the provided search parameters.")
+  listSavedFilter is Operations.ResourceList<SavedFilter, CommonListTrait>;
+  @doc("Retrieve a saved filter by filterName.")
+  getSavedFilter is Operations.ResourceRead<SavedFilter>;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Create or update a saved filter with a given filterName.")
+  @put
+  putSavedFilter is Foundations.ResourceOperation<
```

Comment:
Don't use the http method in the operation name -- describe what the operation does (just like in the description).
```suggestion
  createOrReplaceSavedFilter is Foundations.ResourceOperation<
```